pkg/web/apps/apps_http_handler.go

```diff
@@ -32,3 +33,26 @@ func (a *HTTPHandler) GetApps(c echo.Context) error {
 
 	return c.JSON(http.StatusOK, apps)
 }
+
+// GetAppsId returns apps by id as JSON from the AppService
+func (a *HTTPHandler) GetAppsID(c echo.Context) error {
+
+	id := c.Param("id")
+	if !IsValidUUID(id) {
+		return httperrors.BadRequest(c, "Invalid id supplied")
+	}
+
+	apps, errMessage := a.Service.GetAppsID(id)
+
+	if errMessage == "App Not Found" {
```

Comment:
This will need to be changed once my PR is merged today. It is a very quick change, talk to me if you want to collaborate on it.

Reply:
It's not a huge change , will change the unit tests and the param's in the signature